niceml/dagster/ops/experiment.py

```diff
@@ -39,7 +40,7 @@ def experiment(context: OpExecutionContext) -> ExperimentContext:
         run_id=local_run_id,
         short_id=local_short_id,
     )
-    write_op_config(op_config, exp_context, OpNames.OP_EXPERIMENT.value)
```

Comment:
Mlflow should not be a replacement of writing everything to a experiment folder. But we removed it here because it contained credentials. We solved this in a different way so it is now possible to include it again.

Reply:
If that needs to be removed for security reasons here, that's fine for me.